src/types.rs

```diff
@@ -34,6 +34,11 @@ macro_rules! iden_trait {
 
             fn unquoted(&self, s: &mut dyn fmt::Write);
         }
+
+        /// Identifier
+        pub trait IdenStatic: Iden + Copy + 'static {
```

Comment:
@tyt2y3 why we need `'static` lifetime for `IdentStatic`? I understand why we need it for function result.

Reply:
This basically prevents `Box<dyn IdenStatic>` which SeaORM somehow needs.
Alternatively, we don't align this with SeaORM and so rename this trait to something else.